frontend/src/components/chat/ChannelView.css

```
@@ -333,6 +341,57 @@
+  align-items: center;
+  flex-shrink: 0;
+  width: 12px;
+  height: 12px;
+}
+
+/* AgentBadge declares `color: var(--text-muted)` on the svg itself, which beats
+   the wrapper's inherited color. Opt back into the per-agent tint the wrapper's
+   inline `colorVar` sets, otherwise every presence glyph paints muted. */
+.ch-presence__glyph .agent-badge,
+.ch-presence__glyph .agent-badge--fallback {
+  color: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)